metaflow/package.py

```diff
@@ -46,7 +53,7 @@ def _walk(self, root, exclude_hidden=True):
             for fname in files:
                 if fname[0] == '.':
                     continue
-                if any(fname.endswith(suffix) for suffix in self.suffixes):
+                if any(fname.endswith(suffix) for suffix in suffixes_to_check):
```

Comment:
minor nit - in lieu of L:42-44 we can just have `... in self.suffixes + addl_suffixes` here